src/components/photoGallery/LargeImage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+interface LargeImageProps {
+  largeImageUrl: string;
+}
+
+export const LargeImage: React.FunctionComponent<LargeImageProps> = ({ largeImageUrl }) => {
+  return (
+    <div>
+      <img className="large-picture" src={largeImageUrl} alt="Large Picture" width="90%" />
```

Comment:
The alt text needs to describe the _content_ of the picture, so in your case it should be something like "Earth, as seen from the Moon". But in this component, you don't know which image you're going to be displaying (as you take `largeImageUrl` as a prop) so you need to add the alt text as a prop to this component and pass it down!